test-utils/src/lib.rs

```diff
@@ -165,3 +165,37 @@ pub fn drop_catalog_data<T: Into<String>>(
     }
     Ok(())
 }
+
+/// check_server_version_for_test verifies that the server version is valid if the test specifies min or max server versions.
+fn check_server_version_for_test(
+    min_server_version: &Option<String>,
+    max_server_version: &Option<String>,
+    server_version: &Option<String>,
+) -> bool {
+    // max server version implies there is some version above which the test should not run; hence
+    // if the server version is "latest" we skip the test.
+    if server_version == &Some("latest".to_string()) {
+        return !max_server_version.is_some();
+    }
+    // rapid can be a moving target, so for now, we will skip tests that have version constraints on rapid
+    if server_version == &Some("rapid".to_string()) {
+        return max_server_version.is_some() && min_server_version.is_some();
```

Comment:
Can you clarify this logic.  It should return `true` if these are both not specified?
Wouldn't it then be:
```
    return !min_server_version.is_some() && !max_server_version.is_some();
```

or use `is_none()` instead